compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ImageComposeScene.skikoMain.kt

```diff
@@ -193,8 +193,19 @@ class ImageComposeScene @ExperimentalComposeUiApi constructor(
     /**
      * Returns the current content size
      */
+    @Deprecated("Will be removed in 1.7", replaceWith = ReplaceWith("calculateContentSize()"))
     val contentSize: IntSize get() = scene.contentSize
 
+    /**
+     * Returns the current content size in infinity constraints.
+     *
+     * @throws IllegalStateException when [ComposeScene] content has lazy layouts without maximum size bounds
+     * (e.g. LazyColumn without maximum height).
+     */
+    fun calculateContentSize(): IntSize {
```

Comment:
JFYI I'll mark this as `ExperimentalComposeUiApi` too in my branch

Reply:
If we mark the whole ComposeScene as `@Internal`, then there is no need for that (mark calculateContentSize)

Reply:
@MatkovIvan It's `ImageComposeScene`

Reply:
Ok, make sense there